templates/lib/validatorset.js

```diff
@@ -2,7 +2,7 @@ var Joi = require('joi');
 
 var fieldValidators = {
   username:       Joi.string().alphanum().min(1).required().label('Username'),
-  password:       Joi.string().min(8).required().label('Password'),
+  password:       Joi.string().regex(/^\S{8,128}$/).label('Password'),
```

Comment:
for future-proofing, we probably want `/.../u` here. Mostly because 128 single byte characters is not the same as 128 multibyte characters